starname/chain.json

```diff
@@ -30,21 +30,19 @@
   },
   "codebase": {
     "git_repo": "https://github.com/iov-one/starnamed",
-    "recommended_version": "v0.10.18",
+    "recommended_version": "v0.11.7",
     "compatible_versions": [
-      "v0.10.17",
-      "v0.10.18"
+      "v0.11.7"
     ],
     "genesis": {
       "genesis_url": "https://gist.githubusercontent.com/davepuchyr/6bea7bf369064d118195e9b15ea08a0f/raw/genesis.json"
     },
     "versions": [
       {
-        "name": "v0.10.18",
-        "recommended_version": "v0.10.18",
+        "name": "v0.11.7",
```

Comment:
within the `versions` array, can we please keep v0,10,18 where it is, but then add v0.11.7 as a new object below? We're trying to retain a version history with this